drop_server/backend/views.py

```diff
@@ -107,6 +110,10 @@ def get_if_modified_since(request):
     return (False, datetime.fromtimestamp(0)) if since is None else (True, dateparser.parse(since))
 
 
+def set_last_modified(response, modificationDate):
+    response.headers['Last-Modified'] = modificationDate.strftime("%a, %d %b %Y %H:%M:%S GMT")
```

Comment:
Given UTC itself is a valid timezone...
Problem here is, at least in the test, the `created_at` doesn't know its time zone (neither its name, nor its offset) and thus both %z and %Z are empty leading to an invalid format of the header.
According to [RFC2616](https://www.w3.org/Protocols/rfc2616/rfc2616-sec3.html#sec3.3.1), this is an example of the preferred format:
```
Sun, 06 Nov 1994 08:49:37 GMT  ; RFC 822, updated by RFC 1123
```
and without the `GMT` it is not only unpreferred but invalid.
so we actually need UTC labelled as `GMT`. And I assume it is. But it would be preferrable to ensure that the time offset is UTC+0 (which is impossible because the `created_at` doesn't know its offset (probably my fault but I don't know why)